tsconfig.json

```diff
@@ -25,11 +25,18 @@
     "noFallthroughCasesInSwitch": true,
     "types": [
       "@testing-library/jest-dom"
-    ]
+    ],
+    "paths": {
+      "@casavo/habitat": ["./src/index.ts"],
+      "@casavo/habitat/badge": ["./src/components/Badge"],
```

Comment:
why we have source code with capital letter and import with lowecase?

Reply:
my bad, updated all component import path with pascal case